includes/options/views/class-amp-analytics-options-serializer.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+class Analytics_Options_Serializer {
+
+	public static function save() {
```

Comment:
As discussed, we should look into using the settings API for this instead of manually handling the save (e.g. https://konstantin.blog/2012/the-wordpress-settings-api/). Right now, this handler doesn't have proper capability checks or CSRF protection, which we get "for free" with the settings API.

We can register the setting with `register_setting` and with the `sanitize_callback` where most of the logic from this `save` would move to. Also, thinking to the future, it might be good to nest options under an `amp_settings` option so the structure looks something like:

```
amp_settings => array(
    amp_analytics => array(
        'googleanalytics-basd12' => array( ... ),
        'parsely-sua231' => array( ... ),
    ),
)
```

If we want to keep it as a top-level option for analytics, we need to rename it to `amp_analytics_config` (to avoid conflicts with other plugins/themes).

I'll see if I can pull together an example of moving things over.